testplan/web_ui/testing/src/Common/defaults.js

```diff
@@ -3,6 +3,7 @@
  */
 const GREEN = '#228F1D';
 const RED = '#A2000C';
+const ORANGE = 'orange';
```

Comment:
why is this given as a name when all of the other colours are hex values?

Reply:
> why is this given as a name when all of the other colours are hex values?

I think the named css colour is more readable. It's ok. I will change the value to #FFA500